test/PersistentOrderedMap.test.mo

```diff
@@ -177,6 +177,11 @@ run(
         natMapOps.contains(buildTestMap(), 0),
         M.equals(T.bool(false))
       ),
+      test(
+        "maxEntry",
```

Comment:
Please, add tests for `minEntry` (here and below)

Reply:
fixed